st2client/tests/unit/test_auth.py

```diff
@@ -22,9 +21,10 @@
 import requests
 import argparse
 import logging
+from cStringIO import StringIO
```

Comment:
For cross version compatibility it would be better to use `six.StringIO` (yeah, this one shouldn't cause issues :D)